### PR TITLE
Added credential and related terms

### PR DESCRIPTION
**Pull request by zhengj2007**
Import 'employer role' from OMRSE
Added following terms:
OCCO:00000118	credential, issue: https://github.com/Occupation-Ontology/OccO/issues/31
OCCO:00000119	occupational credential
OCCO:00000120	credential granting agency
OCCO:00000121	competence, issue: https://github.com/Occupation-Ontology/OccO/issues/29
OCCO:00000122	occupational credential holder
OCCO:00000123	certificate
OCCO:00000124	certification
OCCO:00000125	academic degree
OCCO:00000126	license
OCCO:00000127	credential grantor role
OCCO:00000128	credential issuing process
OCCO:00000129	quality assurance group
OCCO:00000130	accredited by

### CODE REVIEW

**Review by Finn1928 — COMMENTED** [review]
The one thing I notice is that content appears to be missing from occo-edit. When I open it, I can't find any of the terms from occo_terms, along with 'document', 'organization', 'employer role', and 'social act'.

**Comment by zhengj2007** [comment]
> The one thing I notice is that content appears to be missing from occo-edit. When I open it, I can't find any of the terms from occo_terms, along with 'document', 'organization', 'employer role', and 'social act'.

These terms are imported from different external ontologies, you can find them in IAO_imports.owl, OBI_imports.owl and OMRSE_imports.owl. Total 12 files including 4 OWL files were changed and 3 new OWL files added.

File: src/ontology/Modules/occo_terms.owl
Comment (giacomodecolle) [inline src/ontology/Modules/occo_terms.owl:543]: Definition reads "A credential that is designed to be about an organism's competence or qualifications with respect to an occupation"

File: src/ontology/Modules/occo_terms.owl
Comment (giacomodecolle) [inline src/ontology/Modules/occo_terms.owl:553]: Label reads "occupational credential"

File: src/ontology/Modules/occo_terms.owl
Comment (giacomodecolle) [inline src/ontology/Modules/occo_terms.owl:614]: Label is "occupational credential holder"

File: src/ontology/Modules/occo_terms.owl
Comment (giacomodecolle) [inline src/ontology/Modules/occo_terms.owl:723]: Definition reads "A social act in which a credential granting agency asserts that an organism has satisfied the
requirements for being awarded a credential, having as output a credential about the organism.

File: src/ontology/Modules/occo_terms.owl
Comment (giacomodecolle) [inline src/ontology/Modules/occo_terms.owl:582]: I don't see this class in the submitted paper

**Review by giacomodecolle — CHANGES_REQUESTED** [review]
Some definitions and labels in the submitted version of the paper are different. When you change these, would you mind also changing terminology in the .tsv file accordingly?

File: src/ontology/Modules/occo_terms.owl
Comment (Finn1928) [reply src/ontology/Modules/occo_terms.owl:582]: Do you mean 'competence'? I see that in the paper. Though the definition given here needs to be adjusted slightly.

File: src/ontology/Modules/occo_terms.owl
Comment (Finn1928) [reply src/ontology/Modules/occo_terms.owl:603]: I think it says 'An organism that an occupation credential is about'

File: src/ontology/Modules/occo_terms.owl
Comment (zhengj2007) [reply src/ontology/Modules/occo_terms.owl:723]: I updated the definitions based on the OccO paper that Finn shared with me. 

The definition of 'occupational credential holder' is 'An organism that an occupation credential is about.'. I did not change its parent from 'Homo sapiens' to 'organism' since we have not imported 'organism' in OccO yet. 
The OBI ['organism'](http://purl.obolibrary.org/obo/OBI_0100026) is equal to 'Viruses' or 'Bacteria' or 'Archaea or Eukaryota'. The CARO ['organism'](http://purl.obolibrary.org/obo/CARO_0001010) is 'Material anatomical entity that is a member of an individual species or is a viral or viroid particle'. Please let me know which 'organism' OccO thought be a 'occupational credential holder'. Then I will update it in the OccO. Thanks!

**Comment by zhengj2007** [comment]
I updated the labels and definitions based on the OccO paper Finn sent to me.
 
The definition of 'occupational credential holder' is 'An organism that an occupation credential is about.'. I did not change its parent from 'Homo sapiens' to 'organism' since we have not imported 'organism' in OccO yet.
The OBI ['organism'](http://purl.obolibrary.org/obo/OBI_0100026) is equal to 'Viruses' or 'Bacteria' or 'Archaea or Eukaryota'. The CARO ['organism'](http://purl.obolibrary.org/obo/CARO_0001010) is 'Material anatomical entity that is a member of an individual species or is a viral or viroid particle'. Please let me know which 'organism' OccO thought be a 'occupational credential holder'. Then I will update it in the OccO.

**Review by Finn1928 — APPROVED** [review]
I'm still unsure of whether the occo-edit is missing terms, but we can see if that's the case after the changes are approved and add them afterwards if they're missing

**Review by giacomodecolle — APPROVED** [review]

**Pull request merged**
